### DISCUSSION TITLE
[Windows] Window framing (no rounded window/borderless)

### DISCUSSION THREAD

**Discussion by supersnellehenk** [f1multiviewer/issue-tracker#44] [Feature requests]
Currently, if you stack two windows like in this screenshot 
![image](https://user-images.githubusercontent.com/25726058/174350963-3482b966-fbcc-4c7b-88ab-11de3817a48d.png)
There's gaps around the players. A lot of Windows apps go back to a rectangular frame to circumvent this design 'issue'.

**Comment by JJWatMyself** [comment]
This may be due to Maintain Aspect Ratio being enabled on each video stream and your display resolution.  If you right click each stream and toggle this to OFF and retry, does the gap disappear?

**Reply by supersnellehenk** [reply]
I can fix the gap in between there if I do disable that, but then I get black bars in my video. Both the streams and my screen are 16:9 and those streams are at half screen.

![image](https://user-images.githubusercontent.com/25726058/174448508-23ad9b69-bf81-45da-a455-6cd1c3179deb.png)

If I manually drag them into each corner, I can make them behave like the browser windows like shown in the next screenshot. 
![image](https://user-images.githubusercontent.com/25726058/174448534-f5b4c493-4021-4d9b-852c-028aaccb1c94.png)

Note: this also makes those two streams act as a group in Windows.

**Comment by f1multiviewer** [comment]
Just wanted to let you know that I'm testing with this. There are some issues (especially with the fullscreening), but I think this can ship soon (not next release though).
